analytics.js

```diff
@@ -44,6 +48,10 @@ import { addDomEventHandlers } from "./utils/autotrack.js";
 import ScriptLoader from "./integrations/ScriptLoader";
 import parseLinker from "./utils/linker";
 import CookieConsentFactory from "./cookieConsent/CookieConsentFactory";
+import * as BugsnagLib from "./metrics/error-report/Bugsnag";
+
+// Load Bugsnag client SDK while SDK initialisation
+BugsnagLib.load();
```

Comment:
Change of plans. Please move the initialization also to the `load` call.

Reply:
Addressed

Reply:
Please move it next to `Bugsnag.init`.

Reply:
Addressed